.github/workflows/JetNews.yaml

```diff
@@ -0,0 +1,113 @@
+name: JetNews
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'JetNews/**'
```

Comment:
I assume you can't use SAMPLE_PATH here yet?

Reply:
Nope, `env` is a runtime thing and these are statically read.

Reply:
I'll add some comments once I've got it working